docs/docs-source/docs/modules/console/pages/index.adoc

```diff
@@ -5,11 +5,8 @@
 
 include::ROOT:partial$include.adoc[]
 
-Available with a Lightbend Platform subscription, Lightbend Console provides deep insights and observability for Cloudflow applications. It helps you verify that your application looks as you expect--with the correct streamlets and the proper connections, each using the desired schema. In the Console, you can watch your application run, check on key performance indicators, such as latency and throughput, and observe the health of each streamlet and the monitors computed on them.
+Available with a Lightbend Platform subscription and on Google Cloud Platform marketplace, Lightbend Console provides deep insights and observability for Cloudflow applications. It helps you verify that your application looks as you expect--with the correct streamlets and the proper connections, each using the desired schema. In the Console, you can watch your application run, check on key performance indicators, such as latency and throughput, and observe the health of each streamlet and the monitors computed on them.
```

Comment:
```suggestion
Available with a Lightbend Platform subscription, Lightbend Console provides deep insights and observability for Cloudflow applications. It helps you verify that your application looks as you expect--with the correct streamlets and the proper connections, each using the desired schema. In the Console, you can watch your application run, check on key performance indicators, such as latency and throughput, and observe the health of each streamlet and the monitors computed on them.
```

Reply:
Will do.

Reply:
I removed the whole console section, since it is in the Enterprise doc and is a commercial feature.